python/cells.cpp

```diff
@@ -610,6 +610,7 @@ void register_cells(pybind11::module& m) {
                 if (cm) d.set_default(arb::membrane_capacitance{*cm});
                 if (rL) d.set_default(arb::axial_resistivity{*rL});
                 if (tempK) d.set_default(arb::temperature_K{*tempK});
+                return d;
```

Comment:
Let's rename `d` `dec` to be consistent with the other method bodies.

Reply:
I am going to defer you to another PR for that. This is doing a lot already.